sacad/sources/coverartarchive.py

```diff
@@ -0,0 +1,57 @@
+"""Cover art archive cover source."""
+
+from sacad import __version__
+from sacad.cover import CoverImageMetadata, CoverSourceResult, CoverSourceQuality
+from sacad.sources.base import CoverSource
+import aiohttp
+import json
+
+
+class CoverArtArchiveSource(CoverSource):
+    """
+    Cover source using the Cover Art Archive official API.
+
+    The documentation of the API is available at
+    https://musicbrainz.org/doc/Cover_Art_Archive/API
+
+    """
+
+    # The cover art archive is tied with Musicbrainz.
+    # Therefore, in order to get a cover, one must first get a release
+    # identifier or release group identifier.
+    # To get the release (or release group) id, one can either read from
+    # the tags (which isn't allowed for now in sacad), or one can ask the
+    # musicbrainz database for identifiers from the album and artist name.
+    # See https://musicbrainz.org/doc/MusicBrainz_API for the API of
+    # musicbrainz.
+    BASE_URL = "https://musicbrainz.org/ws/2"
+    API_KEY = ""
+    API_SECRET = ""
+
+    def getSearchUrl(self, album, artist):
+        return CoverSource.assembleUrl(
+            f"{self.BASE_URL}/release-group", {"query": f"artist:{artist} AND album:{album}", "fmt": "json"}
```

Comment:
I think you may need to quote the artist and album strings, and check that properly handles strings with spaces and/or ' embedded in them.

See also :
https://github.com/desbma/sacad/blob/6e66fe380d3eb604e7f8dda2733bce3402a4e517/sacad/sources/base.py#L211-L221

These are called by default and remove punctuation from the query, maybe this is not desirable for this source since it seem to handle it properly (honest question, I have no idea).